src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <img
```

Comment:
The 'img' elements should be self-closing. You can close it by adding a '/' before the '>' like this: <img />.